src/test/java/dog/pawbook/testutil/TypicalEntities.java

```diff
@@ -128,6 +128,10 @@
     public static final Program GENERAL_KNOWLEDGE = new ProgramBuilder().withName("General Knowledge")
             .withSessions("27-08-2021 09:00").withDogs(10, 12, 14).withTags("Puppies").build();
 
+    // Manually added - Dog's details found in {@code CommandTestUtil}
+    public static final Program BEHAVIOURAL = new ProgramBuilder().withName(VALID_NAME_BEHAVIOURAL)
+            .withSessions(VALID_SESSION_1).withDogs(VALID_DOG_ID_1, VALID_DOG_ID_2).withTags(VALID_TAG_PUPPIES).build();
```

Comment:
you might not want to have dogs enrolled for the manually added programs, since it doesn't make sense for a new program to have any dogs enrolled, perhaps it is better to restrict this default list of dogs to the ones that's part of the typical database